queries.sql

```diff
@@ -61,4 +61,42 @@ SELECT neutered, AVG(escape_attempts) as avg_escapes FROM animals GROUP BY neute
 
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+-- Project 3
+
+SELECT name FROM animals
+JOIN owners ON owners.id = animals.owner_id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
```

Comment:
- [ ] Pokemon is supposed to be the species only for animals that doesn't end with `mon`. once you apply the changes suggested early, you will be good to go. the query here is correct :+1: